packages/lesswrong/components/ea-forum/EAHomeMainContent.tsx

```diff
@@ -256,7 +273,10 @@ const EAHomeMainContent = ({FrontpageNode, classes}:{
   }
   
   const handleTabClick = (tab) => {
-    setActiveTab(tab)
+    history.replace({
+      ...location,
+      search: qs.stringify({...query, tab: tab.slug}),
+    })
```

Comment:
Why `history.replace`? Oh, prolly if you click it a bunch that's an issue. Could you have it push the first time and replace after that? IDK, prolly not worth the effort.

Reply:
Yeah Ollie and I weren't sure what was best, but it didn't seem to matter that much either way, and it's easy to change if we feel like it's not right